engines/native/sandbox.go

```diff
@@ -170,16 +188,18 @@ func (s *sandbox) waitForTermination() {
 
 	s.resolve.Do(func() {
 		// Halt all other sub-processes
-		system.KillByOwner(s.user)
+		if s.engine.config.CreateUser {
+			system.KillByOwner(s.user)
+		}
```

Comment:
This is probably a nvm mind too... because you can am I right? The process is already dead and children have have parent pid 1.